Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -319,6 +319,17 @@ extension BrowserViewController: WKNavigationDelegate {
 
     if navigationAction.targetFrame?.isMainFrame == true,
       BraveSearchManager.isValidURL(url) {
+
+      // Add Brave Search headers if Rewards is enabled
+      if rewards.isEnabled && navigationAction.request.allHTTPHeaderFields?["X-Brave-Ads-Enabled"] == nil {
```

Comment:
Should it be appended in private browsing mode?

Reply:
@kylehickinson ads should not work in Private browsing mode, can you please confirm? Thanks

Reply:
Right now it sends `x-brave-ads-enabled` when in private mode, I think we need to implement an additional check here

Reply:
they don't but the `isEnabled` property does not check for pbm I assume, Kyle to confirm